docs/site/downloads/cldr-48.md

```diff
@@ -220,33 +250,25 @@ For a full listing, see [Delta Data].
 ## Migration
 
 - Number patterns that did not have a specific numberSystem (such as latn or arab) had be deprecated for many releases, and were finally removed.
-- **TBD — add many items!**
+- The default week numbering changes to ISO instead being based on the calendar week starting in CLDR 48 [CLDR-18275]. The calendar week is will be more clearly targeted at matching usage in displayed month calendars.
+- Additionally, language and territory data in `languageData` and `territoryInfo` data received significant updates to improve accuracy and maintainability [CLDR-18087]
+- The likely language for Belarus changed to Russian [CLDR-14479]
+- The pre-Meiji Japanese eras were removed: There was too much uncertainty in the exact values them 
```

Comment:
This needs to move to be a CLDR 49 advanced warning as we have not removed the eras yet https://unicode-org.atlassian.net/browse/CLDR-11400